clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -72,7 +72,13 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
     return [];
   };
 
-  const { currentLocale } = useI18n();
+  const { currentLocale, setCurrentLocale } = useI18n();
```

Comment:
It occurs to me now... is the language switcher only supported on the NoticeOverlay, but not TcfOverlay yet? If so let's track a follow-up ticket to add to TCF as well.

Reply:
ah, good catch. It is all set up to work, but didn't have this context stuff so is likely not working.

Reply:
https://ethyca.atlassian.net/browse/PROD-1892